Essentials/src/main/resources/messages.properties

```diff
@@ -664,11 +909,19 @@ recipeShapeless=\u00a76Combine \u00a7c{0}
 recipeWhere=\u00a76Where\: {0}
 removeCommandDescription=Removes entities in your world.
 removeCommandUsage=/<command> <all|tamed|named|drops|arrows|boats|minecarts|xp|paintings|itemframes|endercrystals|monsters|animals|ambient|mobs|[mobType]> [radius|world]
+removeCommandUsage1=/<command> <mob type> [world]
+removeCommandUsage1Description=Removes all of the given mob type in the current world or another one if specified
+removeCommandUsage2=/<command> <mob type> <radius> [world]
```

Comment:
I don't think it's possible to specify a radius AND a world (the radius just gets set to 0?)

Reply:
It's not but there still needs to be something in that argument so I think the best solution is to leave it there.